tasks/prepare_binaries.js

```diff
@@ -0,0 +1,101 @@
+const path = require('path')
+const fsExtra = require('fs-extra')
+
+const { exec } = require('pkg')
+const puppeteer = require('puppeteer')
+
+const appName = 'asciidoctor-web-pdf'
+const buildPath = 'build'
+
+// Gets chromium browser and builds binaries using pkg
+// Can specify linux/mac/win as first argument to only build one of these platforms
+// NOTE: Intended to be run from the _project root only_
+
+const builds = {
+  linux: { target: 'node10-linux-x64' },
+  mac: { target: 'node10-macos-x64' },
+  win: { target: 'node10-macos-x64', suffix: '.exe', puppeteerName: 'win64' }
+}
```

Comment:
We should probably use Node 12: https://github.com/asciidoctor/asciidoctor-reveal.js/blob/69dc0cb609897a43e112b33f1dd632886160324e/package.json#L22

Node 10 will be EOL in April 2021.